plugins/headless/hu.bme.mit.gamma.headless.server/src/config.properties

```diff
@@ -0,0 +1,5 @@
+%headless.gamma.path=E:/Egyetem/GammaWrapper/HeadlessEclipse/gammaapi/eclipse
+headless.gamma.path=D:/2-Programming/Work/BME2021/MUNKA/openapi_linux_theta/eclipse/eclipse
+%headless.generator.path=E:/Egyetem/GammaWrapper/HeadlessEclipse/generator/eclipse.exe
```

Comment:
Delete the commented lines and add comments for the user stating that these variables should be set to the respective paths of the different artifacts.